config/routes.rb

```diff
@@ -1,4 +1,14 @@
 Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [] do
+        resources :posts, only: [] do
+          resources :comments, only: [:index, :create]
+        end
```

Comment:
- [ ] Dear @SaneMyburg, You have done an impressive job with the routes. But I can see you have not added an `index` route for the `post_index` action. Please update your route as `resources :posts, only: [:index] do` to reference the `index` action of the `post_controller`.